### PR TITLE
[Feature][Zeta] Added other metrics info of multi-table

### LINKED ISSUE #7331: [Feature][Zeta] Added other metrics info of multi-table
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Multi-tables only have count metrics , i tried to perfect this part.

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by corgy-w**
### Purpose of this pull request
issue : #7331 

### Check list

* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hawk9821

**Review by Hisoka-X — COMMENTED** [review]
Please make sure the http api return new metrics too. And add test case in https://github.com/apache/seatunnel/blob/16950a67cd5990ed3510fd91564390e2ab2cd44c/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/MultiTableMetricsIT.java#L99

**Comment by corgy-w** [comment]
> Please make sure the http api return new metrics too. And add test case in
> 
> https://github.com/apache/seatunnel/blob/16950a67cd5990ed3510fd91564390e2ab2cd44c/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/MultiTableMetricsIT.java#L99

get

**Comment by corgy-w** [comment]
PTAL @Hisoka-X @hawk9821 When testing, found that the rate-related metrics could not be aggregated with full precision equality, so need to pay attention to this point.

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/MultiTableMetricsIT.java [hunk 1]
```diff
@@ -82,33 +83,101 @@ public void multiTableMetrics() {
...
+                                    given().get(
+                                                    HOST
+                                                            + instance.getCluster()
+                                                                    .getLocalMember()
+                                                                    .getAddress()
+                                                                    .getPort()
+                                                            + RestConstant.JOB_INFO_URL
+                                                            + "/"
+                                                            + batchJobProxy.getJobId());
+                            // 测试示例中单条 [3, "C", 100] 的数据待大小是13
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/MultiTableMetricsIT.java]: Please change comment to english.

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    private Counter bytes;
+
+    private Map<String, Counter> bytesPerTable = new ConcurrentHashMap<>();
+
+    private Meter bytesPerSeconds;
+
+    private Map<String, Meter> bytesPerSecondsPerTable = new ConcurrentHashMap<>();
+
+    public TaskMetricsCalcContext(
+            MetricsContext metricsContext, String type, boolean isMulti, List<TablePath> tables) {
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]:
```suggestion
            MetricsContext metricsContext, String type, boolean isMulti, List<TablePath> tables) {
```
```suggestion
            MetricsContext metricsContext, PluginType type, boolean isMulti, List<TablePath> tables) {
```

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        }
+    }
+
+    private void initializeMetrics(
+            boolean isMulti,
+            List<TablePath> tables,
+            String sinkWriteCount,
+            String sinkWriteQps,
+            String sinkWriteBytes,
+            String sinkWriteBytesPerSeconds) {
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]:
```suggestion
    private void initializeMetrics(
            boolean isMulti,
            List<TablePath> tables,
            String writeCountName,
            String writeQpsName,
            String writeBytesName,
            String writeBytesPerSecondsName) {
```

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        void process(T t);
+    }
+
+    private String getFullName(TablePath tablePath) {
+        if (StringUtils.isBlank(tablePath.getTableName())) {
+            tablePath =
+                    TablePath.of(tablePath.getDatabaseName(), tablePath.getSchemaName(), "default");
+        }
+        return tablePath.getFullName();
+    }
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]: Already removed in https://github.com/apache/seatunnel/pull/7252/files#diff-88ba3dc2d502d1b50f1b160cb32c6fe40e0e8a74570049cf1a2a05094f4d0ab3

Comment (corgy-w) [reply seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]: not used, forget

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+        }
+    }
+
+    private void initializeMetrics(
+            boolean isMulti,
+            List<TablePath> tables,
+            String writeCountName,
+            String writeQpsName,
+            String writeBytesName,
+            String writeBytesPerSecondsName) {
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]:
```suggestion
    private void initializeMetrics(
            boolean isMulti,
            List<TablePath> tables,
            String countName,
            String qpsName,
            String bytesName,
            String bytesPerSecondsName) {
```
Sorry, I made a mistake.

**Review by Hisoka-X — COMMENTED** [review]
LGTM except one minior problem.

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java [hunk 6]
```diff
@@ -65,12 +67,20 @@
...
 import java.util.Map;
 import java.util.Set;
-import java.util.Spliterators;
 import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
 import static com.hazelcast.internal.ascii.rest.HttpStatusCode.SC_500;
+import static org.apache.seatunnel.api.common.metrics.MetricNames.SINK_WRITE_BYTES;
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java:76]: I think @liugddx better take a look.

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java (hunk 5 above)
Comment (corgy-w) [reply seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/metrics/TaskMetricsCalcContext.java]: done

**Review by Hisoka-X — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
LGTM.Wait for CI to pass

**Pull request merged**